server/methods/accounts/accounts.js

```diff
@@ -356,7 +356,7 @@ export function addressBookAdd(address, accountUserId) {
           "profile.addressBook.$.isShippingDefault": false
         }
       });
-      Hooks.Events.run("afterAccountsUpdate", Meteor.userId(), {
+      Hooks.Events.run("afterAccountsUpdate", userId, {
```

Comment:
I feel like if we are going to do this, we need to change these for every instance where this hook is run. For instance this same hook is called here:

https://github.com/reactioncommerce/reaction/blob/c30d70a962d42285be5367c66b847331f592191c/imports/plugins/included/connectors-shopify/server/methods/import/customers.js#L173

So we need to go through and change not just these but all of them otherwise you will get different info depending on where it is being called.